server/chat-commands/info.ts

```diff
@@ -2198,12 +2288,16 @@ const commands = {
 
 		Config.potd = target;
 		// TODO: support eval in new PM
-		Rooms.PM.eval('Config.potd = \'' + toID(target) + '\'');
+		(Rooms.PM as any).eval('Config.potd = \'' + toID(target) + '\'');
```

Comment:
Um, does this work at all? I would just comment it out if it doesn't...

Reply:
i dont know, we havent used POTD in literal ages

Reply:
Just tested, can confirm it just causes crashes, so i commented it out